src/modules/UI/Wallets/reducer.js

```diff
@@ -182,6 +182,29 @@ export const walletEnabledTokens = (state: any = {}, action: Action) => {
   return state
 }
 
+export const walletLoadingProgress = (state: Object = {}, action: Action) => {
```

Comment:
Please dont use `type: Object`
figure out what the type is exactly
`type { [walletId: string]: number }` or something like this